src/main/java/seedu/address/logic/commands/EditSellerCommand.java

```diff
@@ -36,6 +36,7 @@
 import seedu.address.model.displayable.seller.Seller;
 import seedu.address.model.tag.Tag;
 
+//Solution below adapted by https://github.com/nus-cs2103-AY2324S1/tp
```

Comment:
I don't think you need to give credit for AB3 materials, can check the website for that info